app/src/main/java/com/sun/noteapp/ui/home/NoteVerticalWideAdapter.kt

```diff
@@ -0,0 +1,119 @@
+            super.onBindData(itemData)
+            itemView.apply {
+                textTitleNoteVerticalWide.setBackgroundResource(
+                    ColorPicker.getMediumColor(itemData.color)
+                )
+                layoutContentNoteVerticalWide.setBackgroundResource(
+                    ColorPicker.getLightColor(itemData.color)
+                )
+                textTitleNoteVerticalWide.text = itemData.title
+                val content = if (itemData.type == 1) itemData.content
+                else ConvertString.showActionCheckListByDataString(itemData.content)
+                textContentNoteVerticalWide.text = content
+
+                textRemindTimeNoteVerticalWide.apply {
+                    text = itemData.remindTime
+                    if (itemData.remindTime == Note.NONE) gone() else visible()
+                }
+
+                layoutLabelNoteVerticalWide.apply {
+                    removeAllViewsInLayout()
+                    addLabelView(itemData.label, itemView)
+                    if (itemData.label == Note.NONE) gone() else visible()
+                }
+
+                if (itemData.password != Note.NONE) {
+                    iconLockNoteVerticalWide.visible()
+                    layoutLabelNoteVerticalWide.gone()
+                    textContentNoteVerticalWide.text = EMPTY
+                } else {
+                    iconLockNoteVerticalWide.gone()
+                }
+            }
+        }
+
+        @SuppressLint("SetTextI18n")
+        private fun addLabelView(label: String, view: View) {
+            view.apply {
+                val labels = ConvertString.labelStringDataToLabelList(label)
+                var sum = 0
+                for (i in labels.indices) {
+                    val tv = TextView(context)
+                    val params = LinearLayout.LayoutParams(
+                        ViewGroup.LayoutParams.WRAP_CONTENT
+                        , ViewGroup.LayoutParams.WRAP_CONTENT
+                    )
+                    params.setMargins(5, 0, 5, 0)
+                    tv.apply {
+                        layoutParams = params
+                        text = labels[i]
+                        setBackgroundResource(R.drawable.custom_label)
+                        measure(
+                            ViewGroup.LayoutParams.WRAP_CONTENT,
+                            ViewGroup.LayoutParams.WRAP_CONTENT
+                        )
+                    }
+
+                    sum += tv.measuredWidth + PADDING_WIDTH
+                    if (sum > maxWidthItem - LAST_LABEL_WIDTH) {
+                        if (sum < maxWidthItem && i == labels.size - 1) {
+                            layoutLabelNoteVerticalWide.addView(tv)
+                        } else {
+                            tv.text = "${labels.size - i}+"
+                            layoutLabelNoteVerticalWide.addView(tv)
+                        }
+                        break
```

Comment:
sao lại break ở đây em?

Reply:
nếu vào điều kiện này thì em break luôn vòng for ạ